src/main/java/com/iota/iri/service/API.java

```diff
@@ -1489,15 +1492,18 @@ public static void incEllapsedTimePoW(long ellapsedTime) {
                 transactionViewModels.add(transactionViewModel);
                 prevTransaction = transactionViewModel.getHash();
             } finally {
-                API.incEllapsedTimePoW(System.nanoTime() - startTime);
-                API.incCounterPoW();
-                if ( ( API.getCounterPoW() % 100) == 0 ) {
-                    String sb = "Last 100 PoW consumed "
-                                + API.getEllapsedTimePoW() / 1000000000L
-                                + " seconds processing time.";
-                    log.info(sb);
-                    counter_PoW = 0;
-                    ellapsedTime_PoW = 0L;
+                endTime = System.nanoTime();
```

Comment:
Move the scope of `endTime` here. That is, declare it within `finally` block.

Reply:
Done.